Cart-management/cart-management.html

```diff
@@ -0,0 +1,209 @@
+                                </ul>
+                                <p class="">Prices and delivery costs are not confirmed until you've reached the checkout.</p>
+                                <p>30 days withdrawal and free returns. Read more about <span class="semi-bold">return and refund policy</span>.</p>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+        <footer class=" display-justify-space-around display-align-center">
+            <div class="brand-info">
+
+                <h6>BlueBasket</h6>
+                <p>Copyright © 2022</p>
+                <p>All rights reserved</p>
+                <div class="social-links display-align-center">
+                   <a href="#" class="links"><i class="fa-brands fa-github"></i></a>
```

Comment:
Suggestion to add social links.

Reply:
Thanks for pointing this out, will raise an issue to rectify